src/charm.py

```diff
@@ -92,6 +98,27 @@ def _on_start(self, _) -> None:
         """Ensure that required software is running."""
         pass
 
+    # =======================
+    #  Helpers
+    # =======================
+
+    def _get_mysql_helpers(self):
+        """Returns an instance of the MySQL object from mysqlsh_helpers."""
+        if not self._mysql_helpers:
+            # TODO: replace stubbed arguments once mechanisms to generate them exist
```

Comment:
Add to this TODO that the mechanism must check for credentials existence in the peer relationship data bag.
@shayancanonical , thoughts on this being the nexts tasks?

Reply:
yes! we should definitely tackle the charm related logic next!